apps/web/components/filter/filterSection.tsx

```diff
@@ -0,0 +1,13 @@
+import { ICONS_DATA, Icon } from "../icon";
+
+export const FilterSection = ({icon, filter, children}: {icon: keyof typeof ICONS_DATA, filter: string, children: any}) => (
```

Comment:
Extract props into an interface, extending `PropsWithChildren` from `React` instead of `children: any`.